sdlib/d/gc/emap.d

```diff
@@ -39,7 +39,11 @@ public:
 	}
 
 	void clear(Extent* extent) shared {
-		tree.clearRange(extent.address, extent.size);
+		clear(extent.address, extent.size);
+	}
+
+	void clear(void* address, size_t size) shared {
+		tree.clearRange(address, size);
 	}
```

Comment:
Once again, you got to love how simple this became.

This is however missing a test case.